osidb/models.py

```diff
@@ -2919,6 +2933,45 @@ def validate(self, *args, **kwargs):
         super().clean_fields(*args, **kwargs)
 
 
+class Package(models.Model):
+    """Model representing a package with connected versions."""
+
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    flaw = models.ForeignKey(
+        Flaw,
+        related_name="package_versions",
+        on_delete=models.CASCADE,
+    )
+
+    package = models.CharField(max_length=2058)
```

Comment:
This number caught my eye. If it was `2048` I would not ask but why `2058`? Also those numbers are something tied to the CVEv5 or our estimation of the worst case?

Reply:
@jasinner Do you remember why it was `2058` in the original models, please? I just copied the value. The [schema](https://github.com/CVEProject/cve-schema/blob/master/schema/v5.0/CVE_JSON_5.0_schema.json) specifies `2048`.

Reply:
I think this is just a typo.

Reply:
Would not be `name` or `pkg_name` better than `package`?

Reply:
`package` is the name mandated by the CVE 5.0 schema.

Reply:
I will change `2058` to `2048` today or tomorrow morning while rebasing and squashing the commits.